examples/cephfs/storageclass.yaml

```diff
@@ -20,6 +20,14 @@ parameters:
   # (optional) Ceph pool into which volume data shall be stored
   # pool: cephfs_data
 
+  # (optional) Comma seperated string of Ceph-fuse mount options.
+  # For eg:
+  # fuseMountOptions: debug
+
+  # (optional) Comma seperated string of Cephfs kernel mount options.
+  # Check man mount.ceph for mount options. For eg:
+  # kernelMountOptions: readdir_max_bytes=1048576,norbytes
+
```

Comment:
Also document what `mountOptions` are. It's confusing. It's only for bindfs?

Reply:
I didn't get? You mean i need to define what mount options are? mount options are quite generic term, its also present in the storage class spec.

Reply:
I meant in line #45 there is an another `mountOptions`. What's that for?